test/javascripts/acceptance/search-full-test.js

```diff
@@ -290,6 +290,44 @@ QUnit.test("update status through advanced search ui", async assert => {
   );
 });
 
+QUnit.test(
+  "doesn't update status filter header if wrong value entered through searchbox",
+  async assert => {
+    const statusSelector = selectKit(
+      ".search-advanced-options .select-kit#status"
+    );
+
+    await visit("/search");
+
+    await fillIn(".search-query", "status:none");
+
+    assert.equal(statusSelector.header().label(), "any", 'has "any" populated');
+    assert.equal(
```

Comment:
What is this line testing? On line 302 you entered "status:none" but then never changed it.

Reply:
Our issue was when we enter `status:any_value` then the header of dropdown was displaying `any_value` so to remove that we added this PR, so in this test, I filled `status:none` which is not a value in the dropdown, so it won't change the header of dropdown but before it would display `none` in the dropdown header

Reply:
Yup, and that's what line 304 is testing. It's testing that the value of the dropdown didn't change when you entered an invalid value in the search query.

But this line (305) is checking that the search query is still "status:none", which is always true since you entered that value on line 302 and then never changed it.

Reply:
oooh got it, that's right, ill remove it and add a commit in 5 mins